app/src/main/java/com/github/se/travelpouch/ui/home/TravelList.kt

```diff
@@ -82,75 +105,138 @@ fun TravelListScreen(
   val isPortrait = configuration.orientation == Configuration.ORIENTATION_PORTRAIT
   val mapHeight = if (isPortrait) 300.dp else 200.dp
 
-  Scaffold(
-      modifier = Modifier.testTag("TravelListScreen"),
-      floatingActionButton = {
-        FloatingActionButton(
-            onClick = { navigationActions.navigateTo(Screen.ADD_TRAVEL) },
-            modifier = Modifier.testTag("createTravelFab")) {
-              Icon(imageVector = Icons.Default.Add, contentDescription = "Add")
+  val drawerState = rememberDrawerState(initialValue = DrawerValue.Closed)
+  val scope = rememberCoroutineScope()
+
+  ModalNavigationDrawer(
+      drawerContent = {
+        Box(
+            modifier =
+                Modifier.fillMaxSize()
+                    .testTag("closingMenuBox")
+                    .padding(start = (configuration.screenWidthDp * 0.75).dp)
+                    .clickable {
+                      if (drawerState.isOpen) {
+                        scope.launch { drawerState.close() }
+                      }
+                    }) {
+              FloatingActionButton(
+                  onClick = { scope.launch { drawerState.close() } },
+                  modifier =
+                      Modifier.testTag("closingMenuFab")
+                          .align(Alignment.TopEnd)
+                          .zIndex(1f)
+                          .padding(end = 8.dp)
+                          .padding(top = 8.dp)) {
+                    Icon(imageVector = Icons.Default.Close, contentDescription = "MenuClosing")
+                  }
+            }
+        ModalDrawerSheet(
+            modifier =
+                Modifier.width((configuration.screenWidthDp * 0.75).dp)
```

Comment:
```suggestion
                                Modifier.fillMaxWidth(0.75f)
```
I'd suggest doing this instead, it makes it's easier to read and cleaner.

Reply:
This is done